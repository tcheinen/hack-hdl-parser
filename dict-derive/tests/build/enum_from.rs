use dict_derive::FromPyObject;

#[derive(FromPyObject)]
pub enum Nothing {
    None,
}

fn main() {}
