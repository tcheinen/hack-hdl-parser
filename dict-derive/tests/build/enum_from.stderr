error: Deriving enums is not supported
 --> $DIR/enum_from.rs:4:5
  |
4 | pub enum Nothing {
  |     ^^^^
